core/commands/stat.go

```diff
@@ -18,8 +18,10 @@ import (
 
 var StatsCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "Query IPFS statistics.",
-		ShortDescription: ``,
+		Tagline:          "Query ipfs statistics.",
+		Synopsis:         "ipfs stats <command>",
+		ShortDescription: `'ipfs stats' is a set of commands to help look at statistics for your ipfs node.`,
+		LongDescription: `'ipfs stats' is a set of commands to help look at statistics for your ipfs node.`,
```

Comment:
I think we can probably add some logic that sets the long description to the short description if unset